iOS-sample-app/Showcases/Screen Views/TrackedViewControllers.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+import ContentSquare
+
+// Use this custom class for view controllers in your storyboard, and they will start sending screen views. They
+// will still be editable in the storyboard as usual. You can use the same approach for any cotroller which you edit in
+// a storyboard and for which you don't have code.
+class TrackedViewController: UIViewController {
```

Comment:
I don't want to give that advice. It's a pretty bad idea to start extending every VC you have with this class so that it sends screen views. Yes, it will be a quick fix at first, but as soon as you start needing to extend other stuff because you need other functionality it becomes a mess. I think this is a bad use of inheritance.

It's just a bit more verbose to add `viewDidAppear`s around, but it's much more flexible.